testapi.pm

```diff
@@ -60,11 +60,10 @@ our @EXPORT = qw($realname $username $password $serialdev %cmd %vars
   diag hashed_string
 
   save_tmp_file get_test_data
+
```

Comment:
I guess this can be reverted, no?

Reply:
done. it seems that github is a bit slow and still didn't receive my push